examples/notebooks/balancing-power.ipynb

```diff
@@ -0,0 +1,477 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Implementing balancing power constraints\n",
+    "\n",
+    "## Objective\n",
+    "In this example, we will implement balancing power constraints in PyPSA in a simplified manner. The objective is to enforce some generators to provide balanacing power by running below there nominal capacity but still keep a linear programming problem formulation. \n",

```

Comment:
```suggestion
    "In this example, we will implement balancing power constraints in PyPSA in a simplified manner. The objective is to enforce some generators to provide balanacing power by running below their nominal capacity but still keep a linear programming problem formulation. \n",

```
The start is a bit abrupt. Perhaps clarify what you mean by balancing power in this context (it is used differently in different contexts). Also be consistent with regard to balancing/reserve power. These are different.